src/api/flaskr/service/scenario/block_funcs.py

```
@@ -0,0 +1,392 @@
+from flaskr.service.scenario.utils import get_existing_blocks, get_original_outline_tree
+from flaskr.util import generate_id
+from flaskr.dao import db
+from datetime import datetime
+from flaskr.service.lesson.const import (
+    SCRIPT_TYPE_SYSTEM,
+    STATUS_PUBLISH,
+    STATUS_DRAFT,
+    STATUS_DELETE,
+)
+from flaskr.service.check_risk.funcs import check_text_with_risk_control
+from .utils import change_block_status_to_history, get_original_outline_tree
```

Ruff (0.8.2)

Redefinition of unused `get_original_outline_tree` from line 11

(F811)